include/openenclave/attestation/plugin.h

```diff
@@ -293,16 +291,21 @@ oe_result_t oe_unregister_attester(oe_attester_t* plugin);
  */
 oe_result_t oe_unregister_verifier(oe_verifier_t* plugin);
 
+#if (OE_API_VERSION < 3)
+#define oe_get_evidence oe_get_evidence_v2
+#define oe_get_evidence_v2(p1, p2, p3, p4, p5, p6, p7, p8, p9, p10) \
+    oe_get_evidence_v3(p1, p3, p4, p5, p6, p7, p8, p9, p10)
+#else
+#define oe_get_evidence oe_get_evidence_v3
```

Comment:
@radhikaj Is there a place where we're queuing release notes for v0.9? We'll need to call out the incremented OE_API_VERSION in the notes. `#define OE_API_VERSION 2` will avoid the breakage.